README.md

```diff
@@ -15,10 +15,10 @@ _Disclaimer: Asynction is still at a beta stage. Extensive testing is recommende
   - Event validation (for both ingress and egress events), based on the specified message schemata
   - HTTP request validation, upon connection, based on the channel binding schemata of each namespace
   - Callback validation, upon the ACK of a message, based on the message `x-ack` schemata
+- [Security](#security-authentication-and-authorization) à la [Connexion](https://connexion.readthedocs.io/en/latest/security.html). Handles OAuth2, basic, bearer and API Key based authentication.
```

Comment:
This needs a little more clarification. HTTP (basic & bearer), httpApiKey. There is also a security scheme call APIKey in asyncapi which is different somehow.